_posts/2024-07-01-opensearch-berlin-buzzwords-2024.md

```diff
@@ -0,0 +1,44 @@
+
+We had two talks at this years Buzzwords as well. 
+
+[Praveen Mohan Prasad](https://opensearch.org/community/members/praveen-mohan-prasad.html) and [Hajer Bouafif](https://www.linkedin.com/in/hajerbouafif/) gave a talk on ‘[Rediscover your keyword search: Expand, Enrich and Rewrite](https://program.berlinbuzzwords.de/bbuzz24/talk/WLKNUS/)’, and explained that Dense Vector Search is not the only route to improve your search relevance. They shared how to empower your existing sparse keyword search with semantic search capabilities by leveraging text expansion, metadata enrichment, and query re-writing techniques. Watch the full session: 
+{% include youtube-player.html id=page.rediscoverKeywordSearch %}
+
+[Mingshi Liu](https://opensearch.org/community/members/mingshi-liu.html) and [Owais Kazi](https://opensearch.org/community/members/owais-kazi.html) presented a talk on ‘[Elevating AI Applications with OpenSearch's Flow Framework and RAG Tool](https://program.berlinbuzzwords.de/bbuzz24/talk/SRSS7M/)’ where they introduced OpenSearch's Flow Framework and RAG (Retrieval-Augmented Generation) Tool, which enables developers to create AI-augmented agent, queries, and ingestion flows, integrate ML models, and streamline app development through a no-code/low-code builder. Watch the full session: 
+{% include youtube-player.html id=page.flowFramework %}
+
+*[slides available here](https://github.com/opensearch-project/community/blob/main/presentations/2024-06-11-BerlinBuzzwords-Elevating-AI-Applications-Flow-Framework-RAGTool.pdf)*
+
+* [Full playlist of presentations](https://www.youtube.com/playlist?list=PLq-odUc2x7i8jHpa6PHGzmxfAPEz-c-on)
+* [Photo gallery](https://www.flickr.com/photos/newthinking_de/albums/72177720317853019/)
+
+While the booth was well attended on break, we noticed a few patterns; a lot of the individuals that came by either wanted to congratulate the presenters on how much they got out of the talks, or had questions on their existing workflows and instances, and came by to seek information from the experts in the booth. This told me something; OpenSearch is continuing to be adopted by the community as a platform of choice, and, it gave us ideas on further areas to develop content. If you would like to help contribute that content, check out ‘[help wanted](https://github.com/opensearch-project/project-website/issues?q=is%3Aissue+is%3Aopen+label%3A%22help+wanted%22)’ and ‘[new blog](https://github.com/opensearch-project/project-website/issues?q=is%3Aissue+is%3Aopen+label%3A%22new+blog%22)’ on the project-website repository. Another observation how the booth was packed during breaks in sessions, the booth area as a whole really cleared out during the sessions. This confirms what an incredible show [Plain Schwarz](https://plainschwarz.com/) puts on, and the high value of each presentation given at Berlin Buzzwords is truly appreciated by the community. 
```

Comment:
Do we want to put "project-website" in code font?

Reply:
good call. made that edit as well